### ISSUE TITLE
NVDA does not announce aria-sort="none" in web content

### ISSUE THREAD

**Issue by Wildebrew** [nvaccess/nvda#19387]
### Brief summary

NVDA announces "sorted ascending" for aria-sort="ascending", "sorted descending" for aria-sort="descending", but does not announce aria-sort="none at all

This is revisiting an old discussion I know, but I feel it may require a fresh look.
The aria-sort attribute has 4 token values, ascending, descending, none, and other.
The ascending and descending values of aria-sort are announced by NVDA when applied to a column header in a table, but there is no difference between a column header cell with aria-sort="none" and one with no aria-sort attribute at all.
But there is a big difference, as I understand the aria-sort attribute.
aria-sort="none" means the column is sortable, but no sort is currently applied.
A column header with no aria-sort attribute is not sortable.
This difference is very important when indicating sortable column headers in a table.
Without aria-sort="none" being announced, there is no programmatic way to distinguish a column that is sortable but not sorted from a column that is not sortable at all.
To fill this gap, authors typically add text such as "click to sort" to the column header text, usually as screen reader only text.
This extra text ends up being announced every time a user navigates to a cell in that column, and it is distracting and frustrating.

I admit [the definition of aria-sort="none"](https://www.w3.org/TR/wai-aria-1.2/#aria-sort)
 in the ARIA spec it is somewhat ambiguous, but this seems to be the logical place to start this discussion.

[testing-aria-sort.html](https://github.com/user-attachments/files/24176642/testing-aria-sort.htm)

### Steps to reproduce

1. Open the attached .html file in a browser (Chrome, Edge or Firefox, should not matter).
2. Navigate the column headers (with arrow keys or table navigation)
3. Notice that NVDA does not announce "date of birth" as "sortable, not sorted", in fact it says nothing at all about it

### Actual behavior

NVDA announces the column headers as "Name, sorted ascending", "Date of birth", "phone number"

### Expected behavior

NVDA should announce "Name, sorted ascending", "date of birth, not sorted", "phone number"

### NVDA type

installed copy

### NVDA version

All versions

### Have you tried any other versions of NVDA? If so, please report their behaviors.

                                      Yes, this applies to all NVDA versions I have tested.

### Windows version

N/A

### Name and version of other software in use when reproducing the issue

Tested with latest versions of Chrome, Firefox, Edge.

### Does the issue still occur after restarting your computer?

I have restarted my computer and the issue still occurs

### If NVDA add-ons are disabled, is your problem still occurring?

I do not have any NVDA add-ons installed

### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?

I have not run the COM Registration Fixing Tool

**Comment by gerald-hartig** [comment]
We are following the ARIA spec, and if we made these changes it result in verbose and misleading behaviour from NVDA.
